addons/mail/models/mail_link_preview.py

```diff
@@ -89,37 +89,44 @@ def _is_domain_throttled(self, url):
     @api.model
     def _get_link_preview_from_url(self, url, request_session):
```

Comment:
Shouldn't this be in tools, instead of existing on a model ? We could have "link_preview.py" in "mail/tools" for example, that way we don't have to use a model when trying to preview an url (which is independent from any model).